app/components/provider_interface/application_card_component.html.erb

```diff
@@ -1,4 +1,7 @@
-<section id="<%= dom_id(application_choice) %>" class="app-application-card">
+<section id="<%= dom_id(application_choice) %>" class="app-application-card app-application-card__<%= course_type %>">
+  <% if undergraduate? %>
+    <p class="govuk-body-s app-application-card__course_type">Undergraduate</p>
```

Comment:
I would prefer this text be in a locale.

Reply:
Done